websites/A/Anime Notifier/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "BasicallyWiz",
+    "id": "324588568951390220"
+  },
+  "service": "Anime Notifier",
+  "altnames": [ "notify moe", "notify me", "notifymoe", "notify.moe" ],
+  "description": {
+    "en": "Started as a site to give you notifications whenever your favourite anime goes live, Anime Notifier is now also useable as an anime list, with an extention so you don't have to leave your anime to keep your list up-to-date."
+  },
+  "url": "notify.moe",
+  "version": "0.8.0",
+  "logo": "https://media.notify.moe/images/brand/64.png",
+  "thumbnail": "https://media.notify.moe/images/elements/framed-screenshot-transparent.png",
```

Comment:
only imgur links and min. 512x512 (logo)

Reply:
May I please have a reason for why it's Imgur only? I just thought it would be most efficient to take the logo from the original/official source.

Reply:
Because imgur is more reliable? Just upload it to imgur.